### PR TITLE
Preferred Dishes DB

### PR DESCRIPTION
**Pull request by tanyasingh7**
Created preferred dish db that stores each users selected preferred dishes.

### CODE REVIEW

**Review by Jen-Lopez — DISMISSED** [review]
LGTM! I'd just delete the duplicate test file you got there

**Review by psc358 — APPROVED** [review]

File: front-end/src/pages/PreferredDish.jsx [hunk 1]
```diff
@@ -31,6 +32,21 @@ function PreferredDish() {
     setdishes(response.data);
   };
 
+  const submitDish = async (dish) => {
+    console.log(cookies.get("userID"));
```
Comment (yunko1803) [inline front-end/src/pages/PreferredDish.jsx:36]: delete Console

File: front-end/src/pages/User.jsx [hunk 2]
```diff
@@ -26,8 +26,10 @@ function User() {
     );
 
     if (response.success) {
-      history.push(`/wait`);
+      console.log(response.userID);
```
Comment (yunko1803) [inline front-end/src/pages/User.jsx:29]: remove console

File: back-end/routes/dishes.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const router = express.Router();
+const User = require("../models/user");
+
+router.post("/preferred", function (req, res) {
+
+    const userID = req.body.userID;
+    const dish = req.body.dish;
+    console.log(userID);
+    console.log(dish);
```
Comment (yunko1803) [inline back-end/routes/dishes.js:10]: Is console necessary?

**Review by yunko1803 — COMMENTED** [review]
Slight changes!

**Pull request merged**